docs/updating/from_4.0/to_4.1.md

```diff
@@ -0,0 +1,251 @@
+---
+latest_tag: '4.1.1'
+---
+
+# Update from v4.0.x to v4.1
+
+This update procedure applies if you are using v4.0.latest.
+
+Go through the following steps to update to v4.1.
+
+## Update the app to v4.1
+
+First, run:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer require ibexa/content:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer require ibexa/experience:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer require ibexa/commerce:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+Continue with updating the app:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer recipes:install ibexa/content --force -v
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer recipes:install ibexa/experience --force -v
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer recipes:install ibexa/commerce --force -v
+    ```
+
+The `recipes:install` command installs new YAML configuration files. Look through the old YAML files and move your custom configuration to the relevant new files.
+
+Review the `bundles.php` and leave only third-party entires and entries added by the `recipes:install` command, that start with `Ibexa\Bundle`.
```

Comment:
Are you sure these steps are valid for an update between minor versions? We had them for v3 to v4, but in earlier updates between minor v3 version this was not necessary, AFAIR.

Reply:
I asked Marek about it and he confirmed that recipes:install was indeed necessary. I will go back and double-check.

Reply:
Recipes:install probably, I meant more the "new YAML files" part and modifying bundles.php